topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,220 @@
+>    - {% icon param-file %} *"YAML parameter file"*: `Si-muairss-uep.yaml` (Input dataset)
+>    - *"Does your configuration use the CASTEP calculator?"*: `No`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `Muonated Si-out.cell using Si-muairss-uep.yaml` (Output dataset)
+>
+>    > <comment-title>CASTEP calculator</comment-title>
+>    >
+>    > In our example we have not used the CASTEP calculator. If we had, we
+>    > would have the option of providing a CASTEP parameter file as well.
+>    > However, we would not be able to use the `pm-uep-opt` tool for the next
+>    > step in the workflow, and would have to manually run CASTEP on these
+>    > structures outside of Galaxy, then zip and upload the results to use as
+>    > the input to step 3.
+>    {: .comment}
+>
+{: .hands_on}
+
+The output of this tool is a zipped folder containing muon positions. We can
+then use this with the next tool in our workflow.
+
+## Run UEP optimisation (pm-uep-opt)
+
+Now we have some muonated structures, we have to optimise the position of the
+muon in each. This tool minimises the classical electrostatic forces with
+respect to the muon position, using the {UEP} generated for the original
+structure.
+
+> <hands-on-title>Task description</hands-on-title>
+>
+> 1. {% tool [PyMuonSuite AIRSS UEP Optimise](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/pm_uep_opt/pm_uep_opt/0.2.1+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Muonated structures (.zip)"*: `Muonated Si-out.cell using Si-muairss-uep.yaml` (output of **Generate muonated structures (pm-muairss)** {% icon tool %})
+>    - {% icon param-file %} *"Charge density file (.den_fmt)"*: `Si.den_fmt` (Input dataset)
+>    - {% icon param-file %} *"CASTEP log (.castep)"*: `Si.castep` (Input dataset)
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `UEP results for Muonated Si-out.cell using Si-muairss-uep.yaml` (Output dataset)
+>
+{: .hands_on}
+
+The output of this tool is a zipped folder containing the optimised structures.
+We can then use this with the next tool in our workflow.
+
+## Run clustering for optimised structures (pm-muairss read)
+
+Finally, we can cluster the optimised positions in order to generate a short
+list of potential stopping sites.
+
+> <hands-on-title>Task description</hands-on-title>
+>
+> 1. {% tool [PyMuonSuite AIRSS Cluster](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/pm_muairss_read/pm_muairss_read/0.2.1+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"optimised muonated structures (.zip)"*: `UEP results for Muonated Si-out.cell using Si-muairss-uep.yaml` (output of **Run UEP optimisation (pm-uep-opt)** {% icon tool %})
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `Cluster Report` (Output dataset)
+>    - {% icon param-file %} `Cluster Data` (Output dataset)
+>
```

Comment:
Could add a comment here explaining the different clustering methods, and that we've pre-selected one in the YAML

Reply:
I'll do something for this, but detail will probably depend on how the inclusion of the config tool is handled.